### ISSUE TITLE
Uncaught TypeError: Cannot read property 'dispose' of undefined

### ISSUE THREAD

**Issue by luisrudge** [OmniSharp/omnisharp-atom#363]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.204.0
**System**: Microsoft Windows 10 Pro Insider Preview
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.10.0

### Stack Trace

Uncaught TypeError: Cannot read property 'dispose' of undefined

```
At C:\Users\luis\.atom\packages\omnisharp-atom\lib\omnisharp-atom\features\package-restore.js:26

TypeError: Cannot read property 'dispose' of undefined
  at C:\Users\luis\.atom\packages\omnisharp-atom\lib\omnisharp-atom\features\package-restore.js:26:25
  at Emitter.module.exports.Emitter.emit (C:\Users\luis\AppData\Local\atom\app-0.204.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:82:11)
  at TextEditor.module.exports.TextEditor.destroyed (C:\Users\luis\AppData\Local\atom\app-0.204.0\resources\app.asar\src\text-editor.js:265:27)
  at TextEditor.module.exports.Model.destroy (C:\Users\luis\AppData\Local\atom\app-0.204.0\resources\app.asar\node_modules\theorist\lib\model.js:218:14)
  at Pane.module.exports.Pane.destroyItem (C:\Users\luis\AppData\Local\atom\app-0.204.0\resources\app.asar\src\pane.js:444:18)
  at tabs-tab.<anonymous> (C:\Users\luis\AppData\Local\atom\app-0.204.0\resources\app.asar\node_modules\tabs\lib\tab-bar-view.js:163:24)
  at space-pen-ul.jQuery.event.dispatch (C:\Users\luis\AppData\Local\atom\app-0.204.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4435:9)
  at space-pen-ul.elemData.handle (C:\Users\luis\AppData\Local\atom\app-0.204.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4121:28)

```

### Commands

```
     -0:51.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:50.9.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
  2x -0:50.6.0 editor:move-to-end-of-word (atom-text-editor.editor.is-focused)
     -0:49.8.0 core:move-right (atom-text-editor.editor.is-focused)
     -0:49.6.0 core:paste (atom-text-editor.editor.is-focused)
  2x -0:48.3.0 editor:move-to-end-of-word (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -0:47.6.0 editor:select-to-end-of-word (atom-text-editor.editor.is-focused)
  2x -0:47 core:select-left (atom-text-editor.editor.is-focused)
     -0:46.6.0 core:delete (atom-text-editor.editor.is-focused)
     -0:43.5.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:43.3.0 atom-save-all:saveAll (atom-text-editor.editor.is-focused)
     -0:10.1.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:09.3.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:09.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:08.8.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:08.4.0 core:copy (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "jshint"
    ]
  }
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
atom-beautify, v0.27.10
atom-color-highlight, v3.0.9
atom-save-all, v0.2.0
atom-typescript, v4.5.6
atom-yeoman, v0.3.1
bezier-curve-editor, v0.7.1
color-picker, v2.0.4
editorconfig, v1.0.0
emmet, v2.3.10
keyboard-localization, v1.4.1
language-docker, v1.1.3
less-than-slash, v0.6.0
linter, v0.12.6
omnisharp-atom, v0.10.0
react, v0.11.8
tool-bar, v0.1.5
tool-bar-main, v0.0.8

# Dev
No dev packages
```

**Comment by ferdbold** [comment]
This error seems to happen to me every time I close a .json file

**Comment by david-driscoll** [comment]
A clue! Moat helpful I will look at it tomorrow (might not get time tonight)

-----Original Message-----

**Comment by david-driscoll** [comment]
This should be fixed by #368 if you can let us know.

**Comment by david-driscoll** [comment]
Closing for now, reopen if you can see it again!
